wdfn-server/waterdata/templates/partials/header.html

```diff
@@ -128,7 +128,12 @@
                       <a href="https://www.usgs.gov/mission-areas/water-resources" rel="noopener">Water Resources</a>
                   </li>
                   <li class="usa-nav__secondary-item">
-                      <a href="https://dashboard.waterdata.usgs.gov/" rel="noopener">Water Dashboard</a>
+                      {% if stations|length == 1 %}
+                        <a href="https://dashboard.waterdata.usgs.gov/app/nwd/?aoi={{ stations[0].agency_cd|lower }}-{{ stations[0].site_no }}"
```

Comment:
Did we want to open the page in a new tab? In the user test, Megan mentioned that she thought it should open a new tab, which is sort of the standard, when opening links to a different website. And that website is pretty different.

Reply:
It wasn't part of the ticket and that's how we were doing it. I'm going to merge. We can ask Shawn tomorrow. Easy to fix if needed.